processes/executive-copy-creation-checklist.md

```diff
@@ -47,6 +47,14 @@
 - [ ] Inform the reviewer that the executive copy is ready to review
 - [ ] Commit changes, and create a PR to the Community GitHub repository.
 
+## Atlas Active Element Updates
+
+- [ ] Check if any Atlas active elements will need to be updated if the executive proposal passes.
```

Comment:
Nit: This may be confusing because "For each item, request review of the PR" may be read as suggesting one PR per item.